README.md

```diff
@@ -65,9 +66,14 @@ Embed the [Highlight.js script](https://highlightjs.org/download/) along with on
 <link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin-frappe.css">
 <link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin-macchiato.css">
 <link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin-mocha.css">
-<!-- variable version -->
+<!-- Variable version -->
 <link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.css">
+<!-- Variable version with !important (higher specificity) -->
 <link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.important.css">
+<!-- RGB variable version -->
+<link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.any.css">
+<!-- RGB variable version with !important (higher specificity) -->
+<link rel="stylesheet" href="//unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.any.important.css">
```

Comment:
I think you need to update the links here to match the new files you've created.

Reply:
Well :thinking: hmmmm, I guess it would be incorrect for the same version

Reply:
In any case, the links do need to be updated. I think it's fine pointing to the same version right now since it'll be updated in the release please PR